whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/common/EndUserAbuseCValidator.java

```diff
@@ -0,0 +1,70 @@
+package net.ripe.db.whois.update.handler.validator.common;
+
+import com.google.common.collect.ImmutableList;
+import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.domain.CIString;
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.common.rpsl.attrs.OrgType;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.PreparedUpdate;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import net.ripe.db.whois.update.domain.UpdateMessages;
+import net.ripe.db.whois.update.handler.validator.BusinessRuleValidator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import static net.ripe.db.whois.common.rpsl.ObjectType.ROLE;
+
+@Component
+public class EndUserAbuseCValidator implements BusinessRuleValidator {
+
+    private static final ImmutableList<Action> ACTIONS = ImmutableList.of(Action.CREATE, Action.MODIFY);
+    private static final ImmutableList<ObjectType> TYPES = ImmutableList.of(ObjectType.INETNUM, ObjectType.INET6NUM, ObjectType.AUT_NUM);
+
+    private final RpslObjectDao rpslObjectDao;
+
+    @Autowired
+    public EndUserAbuseCValidator(final RpslObjectDao rpslObjectDao) {
+        this.rpslObjectDao = rpslObjectDao;
+    }
+
+    @Override
+    public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
```

Comment:
Do we need to check the inetnum STATUS? or are *all* status values included? 

The Abuse-c Validation policy [ripe-705](https://www.ripe.net/publications/docs/ripe-705) only refers to direct allocated inetnums needing abuse-c, and we only check certain end user inetnum statuses in our abuse-c validation code.